src/main/java/org/prebid/server/bidder/ix/model/request/IxDiag.java

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.bidder.ix.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+@Value(staticConstructor = "of")
+public class IxDiag {
+
+    String pbsv;
+
+    String pbjsv;
```

Comment:
imho it's better to use JsonProperty on the fields, because those fields names are meaningless themselves and it might be a risk of renaming, that will lead to the unwanted contract changes, so I suggest decreasing such possibilities